src/test/java/org/folio/rest/impl/CurrentFiscalYearTest.java

```diff
@@ -0,0 +1,102 @@
+package org.folio.rest.impl;
+
+import io.vertx.core.json.JsonObject;
+import org.folio.rest.jaxrs.model.Errors;
+import org.folio.rest.jaxrs.model.FiscalYear;
+import org.folio.rest.jaxrs.model.Ledger;
+import org.folio.rest.jaxrs.resource.FinanceLedgers;
+import org.folio.rest.util.HelperUtils;
+import org.junit.jupiter.api.Test;
+
+import java.util.Date;
+import java.util.UUID;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+import static javax.ws.rs.core.Response.Status.NOT_FOUND;
+import static javax.ws.rs.core.Response.Status.OK;
+import static org.folio.rest.util.ErrorCodes.FISCAL_YEARS_NOT_FOUND;
+import static org.folio.rest.util.MockServer.addMockEntry;
+import static org.folio.rest.util.TestEntities.FISCAL_YEAR;
+import static org.folio.rest.util.TestEntities.LEDGER;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+public class CurrentFiscalYearTest extends ApiTestBase {
+
+  @Test
+  public void testOneFiscalYear() {
+
+    FiscalYear year = new FiscalYear().withId(UUID.randomUUID().toString());
+
+    String ledgerId = UUID.randomUUID().toString();
+    Ledger ledger = new Ledger().withId(ledgerId).withFiscalYearOneId(year.getId());
+    addMockEntry(LEDGER.name(), JsonObject.mapFrom(ledger));
+    addMockEntry(FISCAL_YEAR.name(), JsonObject.mapFrom(year));
+
+    FiscalYear response = verifyGet(getCurrentFiscalYearEndpoint(ledgerId), APPLICATION_JSON, OK.getStatusCode()).as(FiscalYear.class);
+    assertThat(response.getId(), is(year.getId()));
+  }
+
+  @Test
+  public void testTwoOverlappedFiscalYears() {
+
```

Comment:
Some description of what this test does will helpful. Something like `logger.info` the way we do in other Test classes. This applies to all tests in this file.